### DISCUSSION TITLE
How to setup about me in introduction plugin

### DISCUSSION THREAD

**Discussion by karthikPattabhiram** [lowlighter/metrics#1116] [Q&A] [answered]
I am unable to setup about me in introduction plugin

**Comment by lowlighter** [comment, marked as answer]
Hi 👋 !

The `introduction` plugin basically show again what you put in your profile bio, when you edit your profile on your user page:

![image](https://user-images.githubusercontent.com/22963968/176346752-a5c696a9-ce0b-4a98-832b-ebda44e278ee.png)
